src/Microsoft.Data.SqlClient/netfx/src/Microsoft.Data.SqlClient.csproj

```diff
@@ -288,6 +288,10 @@
     <Compile Include="..\..\src\Microsoft\Data\SqlClient\SqlColumnEncryptionKeyStoreProvider.cs">
       <Link>Microsoft\Data\SqlClient\SqlColumnEncryptionKeyStoreProvider.cs</Link>
     </Compile>
+    <Compile Include="..\..\src\Microsoft\Data\SqlClient\SqlCommandBuilder.cs">
+      <Link>Microsoft\Data\SqlClient\SqlCommandBuilder.cs</Link>
+      <SubType>Component</SubType>
```

Comment:
I'm actually not sure why it's added.  It's automatically added by visual studios whenever build/save the project, so I end up leaving it there.

Reply:
Nevermind, it looks like I am able to remove it and it doesn't get automatically added back after build now.

Reply:
It does add it back in and I remember fighting against VS re-adding it before and we found some way to prevent it happening. I'll try to find the PR and see if we need to do the same thing on netfx

Reply:
That explains why I was able to remove it this time because `[DesignerCategory()]` was there on line 19